keps/sig-node/1769-memory-manager/README.md

```diff
@@ -636,8 +636,9 @@ fields of API types, flags, etc.?**
 _This section must be completed when targeting beta graduation to a release._
 
 ###### How can an operator determine if the feature is in use by workloads?
-
-  The memory manager data will be available under pod resources API. When it configured with the static policy
+  In order for workloads to request exclusive memory allocation, the pod QoS must be "guaranteed".
```

Comment:
This is not something that operator can control and is users responsibility so this addition doesn't seem right.

Reply:
Agreed. Removed, left only references to metrics and podresources API.